libs/blocks/nav-marquee/nav-marquee.css

```diff
@@ -0,0 +1,275 @@
+  flex: 1 1 auto;
+}
+
+.nav-marquee .toc-item a {
+  color: var(--text-color);
+  outline: none;
+}
+
+.nav-marquee .toc-item a:hover {
+  text-decoration: none;
+}
+
+.nav-marquee .toc-link-text p:last-child {
+  margin-bottom: 0;
+}
+
+.nav-marquee .section-title {
+  font-weight: bold;
+  font-size: var(--type-body-s-size);
+  line-height: 20px;
+}
+
+.nav-marquee .section-description {
+  font-size: 14px;
+}
+
+.nav-marquee .toc-arrow {
+  align-self: center;
+  padding-left: 8px;
+}
+
+.nav-marquee .marquee {
+  min-height: unset;
+}
+
+.nav-marquee .marquee .foreground {
+  width: var(--grid-container-width);
+  margin: 0 auto;
+  padding-top: 56px;
+  padding-bottom: 0px;
+}
+
+.nav-marquee .marquee .foreground .media {
+  width: 100%;
+}
+
+@media screen and (min-width: 600px) {
+  .nav-marquee .content-table .toc-list {
+     flex-flow: row wrap;
+  }
+
+  .nav-marquee .toc-item {
+    flex: 1 1 45%; 
+    padding: 16px;
+  }
+
+  .nav-marquee .marquee .foreground .text h2 {
+    font-size: var(--type-heading-xxl-size);
+    clear: both;
+  }
+
+  .nav-marquee .marquee .foreground .text h3 {
+    font-size: var(--type-heading-m-size);
+    text-transform: none;
+  }
+
+  .nav-marquee .marquee .foreground .media {
+    margin: 0;
+  }
+}
+@media screen and (max-width: 600px) {
+  .nav-marquee .toc-list {
+    gap: 16px;
+  }
+
+  .nav-marquee .toc-item {
+    padding: 16px;
+  }
+
+  .nav-marquee .section-description {
+    font-size: 12px;
+    margin-top: 16px;
+  }
+
+  .nav-marquee .toc-footer{
+    font-size: 12px;
+  }
+
+  .nav-marquee .marquee .foreground {
+    padding-top: 48px;
+  }
+}
+
+@media screen and (min-width: 1200px) {
+  .section .nav-marquee {
+      display: flex;
+      flex-direction: row;
+  }
+
+  .nav-marquee .content-table {
+    background-color: #FFF;
+    border: none;
+  }
+
+  .nav-marquee .section-title {
+    line-height: 22.5px;
+    margin-bottom: 16px;
+    font-size: var(--type-body-m-size);
+  }
+
+  .section .nav-marquee > div {
+    flex: 1 1 auto;
+    background-color: transparent;
+    min-width: 400px;
+  }
+
+  .nav-marquee .marquee .foreground {
+    margin-right: 0;
+    align-content: space-between;
+    flex-flow: row wrap;
+    gap: 0;
+  }
+
+  .nav-marquee .toc-container {
+    padding: 32px;
+    box-sizing: border-box;
+    margin-left: 0;
+    padding-right: 0;
+    background-color: #FFF;
+    margin-top: var(--spacing-xl);
+  }
+
+  .nav-marquee .toc-list {
+    flex-flow: row wrap;
+  }
+  
+  .nav-marquee .toc-container .toc-list {
+    display: flex;
+    flex-direction: column;
+    gap: 24px;
+    margin-top: 24px;
+    margin-bottom: 0;
+    padding: 0;
+  }
+  
+  .nav-marquee .toc-container .toc-item {
+    min-width: 200px;
+    display: flex;
+    position: relative;
+    justify-content: space-between;
+    background-color: #FFF;
+    border: 1px solid #4B4B4B;
+    border-radius: 4px;
+    padding: 24px;
+    cursor: pointer;
+    line-height: 1.5;
```

Comment:
you have a similar definition for `.nav-marquee .toc-item`. The previous one would get inherited on min 1200px
https://github.com/adobecom/milo/pull/786/files#diff-3052973bc600cc270df3e74fc90fefef74cfa0de6424aea73b1b415ea91e58a7R46

Also, you're setting `line-height: 1.5` for the whole section, but then you specify different line-height for the `section-title` which is a child of `toc-item`. Maybe you can add specific line height directly to elements.